data/en/lms_2.json

```diff
@@ -1733,15 +1924,15 @@
                         "questions": [{
                             "id": "understand-welsh-question",
                             "titles": [{
-                                    "value": "Can <em>{{[answers['first-name'][group_instance], answers['last-name'][group_instance]] | format_household_name}}</em>...",
+                                    "value": "Can <em>{{[answers['first-name'][group_instance], answers['last-name'][group_instance]] | format_household_name}}</em> understand, speak, read or write Welsh?",
                                     "when": [{
                                         "id": "proxy-check-answer",
                                         "condition": "equals",
                                         "value": "proxy"
                                     }]
                                 },
                                 {
-                                    "value": "Can you..."
+                                    "value": "Can you understand, speak, read or write Welsh?"
```

Comment:
We could make this conditional on the survey being in welsh. e.g. don't ask it if they're answering in welsh.